R/readTechCosts.R

```diff
@@ -59,29 +79,470 @@ readTechCosts <- function(subtype = "PowerAndHeat") { # nolint
 
 
   if (subtype == "PowerAndHeat") {
-    x <- read.csv("power_and_heat_energyf.csv")
-    names(x) <- c("tech", "value", "year", "variable", "unit")
-    x <- filter(x, x[["tech"]] != "Refurbishment of existing nuclear reactors")
-    x[["value"]] <- as.numeric(x[["value"]])
-    x <- as.quitte(x)
+
+      df <- read_excel("REF2020_Technology Assumptions_Energy.xlsx", sheet = "Power&Heat", range = "A2:V80")

```

Comment:
function `read_excel` is not imported and causes `readTechCosts` to crash each time it's called. suggested change: add `read_excel` the `ImportFrom` fields in the function documentation

Reply:
οκ